azure-pipelines.yml

```diff
@@ -1,39 +1,36 @@
-# ASP.NET
-# Build and test ASP.NET projects.
-# Add steps that publish symbols, save build artifacts, deploy, and more:
-# https://docs.microsoft.com/azure/devops/pipelines/apps/aspnet/build-aspnet-4
-#trigger on master pull
+#trigger on master push
 trigger:
   branches:
     include:
     - main
 pool:
-  vmImage: 'ubuntu-latest'
+  vmImage: ubuntu-latest
 
+# build a docker file
 stages:
   - stage: build
     jobs:
     - job: build
       variables:
-        buildConfiguration: 'Release'
+        buildConfiguration: Release
         directory: 'FakebookNotifications/FakebookNotifications.WebApi'
       steps:
       - task: UseDotNet@2
-        displayName: dotnet sdk 5.x
+        displayName: 'Use sdk 5.x'
         inputs:
-          packageType: 'sdk'
-          version: '5.x'
+          packageType: sdk
+          version: 5.x
           installationPath: $(Agent.ToolsDirectory)/dotnet
-          workingDirectory: '$(directory)'
+          workingDirectory: $(directory)
       # build and publish app
       - script: dotnet build --configuration $(buildConfiguration)
         displayName: dotnet build
-        workingDirectory: '$(directory)/'
+        workingDirectory: $(directory)
       - script: dotnet publish --configuration $(buildConfiguration) --output publish
         displayName: dotnet publish
-        workingDirectory: '$(directory)'
+        workingDirectory: $(directory)
       # publish artifact for eventual release
-      - publish: '$(directory)/publish'
+      - publish: $(directory)/publish
         artifact: app
```

Comment:
now that the dockerfile is in main, this job should just run docker build instead of all this.